docs/governance/overview.md

```diff
@@ -2,29 +2,29 @@
 
-Decred's on-chain governance is supplemented by *Politeia proposal voting*, which doesn't happen directly on-chain but is woven into the Decred blockchain in some ways. 
+If a vote to change the consensus rules is occurring, tickets can also [vote](consensus-rule-voting/overview.md) to approve or reject the proposed change. A proposed change must be approved by 75% of non-abstaining tickets to take effect.
 
-Politeia proposals concern the direction of the project, they may involve the spending of [Decred Treasury funds](http://explorer.dcrdata.org/address/Dcur2mcGjmENx4DhNqDctW5wJCVyT3Qeqkx) (10% of the block reward goes into the Treasury to support development of the project) or amending the [Decred Constitution](decred-constitution.md) or other policies.
 
-Politeia is built around the concept of _transparent censorship_, using [dcrtime](https://github.com/decred/dcrtime). Users cannot be silently censored, they can prove that censorship has occurred.
+## Off-chain voting
 
-The [Politeia web platform](https://proposals.decred.org/) is a reddit-style space to facilitate submitting, viewing and discussing proposals.
+Decisions concerning the direction of the project generally, such as how to spend Treasury funds, or amending the Decred Constitution or other policies, are made via proposals on [Politeia](https://proposals.decred.org/).
 
-Politeia proposals are approved/rejected through "snap" voting. When proposals move to a vote, all live tickets at that moment are eligible to vote Yes/No on the proposal while voting remains open (one week period). Ticket-holders vote through their wallet.
+Anyone may submit a proposal, and all data on Politeia (proposals, comments, upvotes/downvotes) is periodically anchored into the Decred blockchain, using [dcrtime](https://github.com/decred/dcrtime). This enables users to cryptographically prove if censorship has occurred. In this way, Politeia is built around the idea of _transparent censorship_. Users cannot be silently censored. 
```

Comment:
Link to the censorship page here maybe?

Reply:
Good idea. Linked.

Reply:
Link should include the `.md` file extension so it can be verified by mkdocs.

Reply:
Fixed